demo/image_classification/image_provider.py

```diff
@@ -58,24 +58,26 @@ def hook(settings, img_size, mean_img_size, num_classes, color, meta, use_jpeg,
     settings.logger.info('DataProvider Initialization finished')
 
 
-@provider(init_hook=hook)
-def processData(settings, file_name):
+@provider(init_hook=hook, min_pool_size=0)
```

Comment:
`min_pool_size=0` will cause the dataset is not shuffled fully. See [here](https://github.com/baidu/Paddle/blob/develop/doc_cn/faq/index.rst#dataprovider)